assets/out

```diff
@@ -112,6 +112,14 @@ elif [ -n "$build" ]; then
   fi
 
   expanded_build_args=""
+
+  # Propagate proxy settings from Concourse worker environment to image builder
+  for build_arg_name in http_proxy https_proxy no_proxy ; do
+    if [ -n "${!build_arg_name}" ]; then
+      expanded_build_args="$expanded_build_args --build-arg ${build_arg_name}=${!build_arg_name}"
+    fi
+  done
+
   if [ -n "$build_args" ]; then
     expanded_build_args=$(echo $build_args|jq -r 'with_entries(.key |= "--build-arg " + . )|with_entries(.key = .key + "=" +.value)|keys|join(" ")')
```

Comment:
This line would squash your appended values. This wouldn't break any existing pipelines, so it is backwards compatible, but wouldn't you want the proxy values always?

Reply:
Oh, didn't notice that! Thanks.
I've fixed that now. I hope.